core/controllers/creator_dashboard.py

```diff
@@ -77,6 +85,17 @@ class CreatorDashboardHandler(base.BaseHandler):
     """Provides data for the user's creator dashboard page."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'display_preference': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
```

Comment:
Can we limit this only to values in `ALLOWED_CREATOR_DASHBOARD_DISPLAY_PREFS`? Similar in other places where you added schema, even outside this PR.

Reply:
Done

Reply:
Did you look also in other places outside this PR?

Reply:
Yes, considering the case for ```display_preference``` it is only used in creator_dashboard.
Although I have looked in previous PR's for similar arguments but not find anything, since most of my work for adding schemas is part of second milestone I will take care of this condition.
Thank you